src/pollyCLI/pollyintegration.cpp

```diff
@@ -22,26 +29,92 @@ PollyIntegration::PollyIntegration(): nodePath(""), jsPath("")
       QString binDir = qApp->applicationDirPath() + QDir::separator() + ".." + QDir::separator() + ".." + QDir::separator() + ".." + QDir::separator();
       if(!QStandardPaths::findExecutable("node", QStringList() << binDir + "node_bin" + QDir::separator() ).isEmpty())
         nodePath = binDir + "node_bin" + QDir::separator() + "node";
+    #endif
+
+     _dlManager->setRequest("https://raw.githubusercontent.com/ElucidataInc/polly-cli/master/index.js", this);
```

Comment:
Change from 
`_dlManager->setRequest("https://raw.githubusercontent.com/ElucidataInc/polly-cli/master/index.js", this)`
to
`_dlManager->setRequest("https://raw.githubusercontent.com/ElucidataInc/polly-cli/master/prod/index.js", this)`

Reference: https://github.com/ElucidataInc/polly-cli/pull/4

Reply:
Okay, will do

Reply:
@sahil21 
The above url is nor working